frontend/packages/ux-editor/src/components/config/editModal/EditOptions/EditCodeList/EditCodeList.tsx

```diff
@@ -31,58 +35,130 @@ export function EditCodeList<T extends SelectionComponentType>({
     });
   };
 
+  const handleUpload = (file: File) => {
+    uploadOptionList(file, {
+      onSuccess: () => {
+        handleOptionsIdChange(removeExtension(file.name));
+        toast.success(t('ux_editor.modal_properties_code_list_upload_success'));
+      },
+    });
+  };
+
+  const onSubmit = (file: File) => {
+    const fileNameError = findFileNameError(optionListIds, file.name);
+    if (fileNameError) {
+      handleInvalidFileName(file, fileNameError);
+    } else {
+      handleUpload(file);
+    }
+  };
+
+  const handleInvalidFileName = (file: File, fileNameError: FileNameError) => {
+    if (fileNameError == 'invalidFileName') {
+      alert(t('ux_editor.model_properties_code_list_filename_error'));
+    }
+
+    if (fileNameError == 'fileExists') {
+      toast.error(t('ux_editor.modal_properties_code_list_upload_duplicate_error'));
+    }
```

Comment:
Er det noen grunn til å bruke `alert` på den ene feilmeldingen og `toast` på den andre?

Småpirk: Siden du sjekker mot samme variabel, kan du bruke `switch`:
```suggestion
    switch (fileNameError) {
      case 'invalidFileName':
        return alert(t('ux_editor.model_properties_code_list_filename_error'));
      case 'fileExists':
        return toast.error(t('ux_editor.modal_properties_code_list_upload_duplicate_error'));
    }
```

Reply:
Egentlig ikke, toast kan brukes for begge siden det bare er info og ikke et valg for brukeren.